docs/TroubleShooting.md

```diff
@@ -54,6 +54,17 @@ make
 sudo make install
 ```
 
+#### Build break with toolchain
```

Comment:
Can you put this under a new heading
Issues with toolchain ?

Reply:
@abhishek-samsung 
you can move this info out of "Issues on Kconfig-frontend" section,

Reply:
Done